moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -442,9 +447,10 @@ const ProjectsListViewTable = ({ query }) => {
 
   const columnsToReturn = Object.keys(PROJECT_LIST_VIEW_QUERY_CONFIG.columns);
 
-  const { query: projectListViewQuery } = useGetProjectListView({
+  const { query: projectListViewQuery, exportQuery } = useGetProjectListView({
```

Comment:
This hook now builds both a query with current filters, order, and pagination and also an export query that doesn't include the pagination parameters. We can adjust how this works in the future, but this is what we do in prod right now. Also open to splitting out the the export and non-export query building if anyone feels like it makes sense.